src/client/components/Footer/Footer.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import './footer.css'
+
+export const Footer = () => {
+
+    return <footer>
+        <div className='footer'>
```

Comment:
You have a `<footer>` tag and a` <div className='footer'>` tag, both are performing the same function. You can keep only one as 2 seems a bit unnecessary.

Reply:
You are right I fixed it